### PR TITLE
Added Thumbnail component.

### PR DESCRIPTION
**Pull request by benjaminnaesen**

### CODE REVIEW

File: src/components/Thumbnail/Thumbnail.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+	handleImageLoaded() {
+		this.setState({ loaded: true });
+	}
+
+	render() {
+		const { category, src, alt, label, meta }: ThumbnailProps = this.props;
+		const { loaded }: ThumbnailState = this.state;
+
+		const metaClass = loaded ? 'c-thumbnail-meta--img-is-loaded' : '';
```
Comment (e2o) [inline src/components/Thumbnail/Thumbnail.tsx]: Gebruiks `classnames`-pkg

File: src/components/Thumbnail/Thumbnail.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	alt?: string;
+	label?: string;
+	meta?: string;
+}
+
+export interface ThumbnailState {
+	loaded: boolean;
+}
+
+export class Thumbnail extends React.Component<ThumbnailProps, ThumbnailState> {
```
Comment (e2o) [inline src/components/Thumbnail/Thumbnail.tsx]: Graag omzetten naar functional component

File: src/components/Thumbnail/Thumbnail.test.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+			<Thumbnail meta="4 items" category="collection" label="collectie" />
+		);
+
+		const imageContainer = thumbnailComponent.find('.c-thumbnail-image');
+		expect(imageContainer).toHaveLength(0);
+
+		const placeholderElement = thumbnailComponent.find('.c-thumbnail-placeholder');
+		expect(placeholderElement).toHaveLength(1);
+	});
+});
```
Comment (e2o) [inline src/components/Thumbnail/Thumbnail.test.tsx]: - 'Should set the correct className for each category'
- 'Should not render image if `src` was not passed'
- 'Should correctly set `alt` if it was passed'
- 'Should correctly render meta if it was passed'
- 'Should not render meta if it was not passed'

File: src/components/Thumbnail/Thumbnail.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	category,
+	src,
+	alt,
+	label,
+	meta,
+}: ThumbnailProps) => {
+	const [loaded, setLoaded] = useState(false);
+
+	return (
+		<div className={`c-thumbnail c-thumbnail-media c-thumbnail-media--${category}`}>
```
Comment (e2o) [inline src/components/Thumbnail/Thumbnail.tsx]: `classnames` gebruiken

File: src/components/Thumbnail/Thumbnail.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```
Comment (e2o) [inline src/components/Thumbnail/Thumbnail.stories.tsx]: story content misschien in het Engels houden? 😅

File: src/components/Thumbnail/Thumbnail.test.tsx (hunk 3 above)
Comment (benjaminnaesen) [reply src/components/Thumbnail/Thumbnail.test.tsx]: 'Should not render image if src was not passed' is already done in the placeholder render test. I'll fix the rest. 👌

Comment (e2o) [reply src/components/Thumbnail/Thumbnail.test.tsx]: Ja, maar ge gebruikt hier geen Thumbnail component dus dan moet ge dat opnieuw testen hé

**Pull request merged**